pkg/cmd/cli/restore/create_test.go

```diff
@@ -72,7 +71,7 @@ func TestCreateCommand(t *testing.T) {
 		scheduleName := "schedule1"
 		restoreVolumes := "true"
 		preserveNodePorts := "true"
-		labels := "c=foo,b=woo"
+		labels := "b=woo,c=foo"
```

Comment:
The underlying type of the `option.Labels` is a map, iterating the map is in random order, so I guess this is the reason that the unit test may fail sometimes. Could you double-check?

Reply:
`The iteration order over maps is not specified and is not guaranteed to be the same from one iteration to the next.`